mautic/mautic
# Single landing page scripts

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | features
| Bug fix?                               | no
| New feature?                           | yes
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | yes
| Related user documentation PR URL      | N/A
| Related developer documentation PR URL | N/A
| Issue(s) addressed                     | N/A




#### Description:

Added two new fields for landing page:
* Script inside head
* Script inside footer

Anyone with this feature can add custom script to single landing page, and can decide whether to upload it if inside head tag or at the end of body.

![singlelandingscripts](https://user-images.githubusercontent.com/50916237/109322676-dcae6480-7852-11eb-8c95-7c68de8c03df.png)




#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create a landing page
3. Edit the two new inputs with a custom script (example `<script>console.log("Hello world");</script>`)
4. Go to the landing page and check if script is loaded




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/9695"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
